sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -174,143 +174,226 @@
 
 
 ###############################################################################
+###########################   Creating Universes   ############################
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating universes...')
+
+universes = list()
+universes.append(Universe(name='uo2 pin cell'))
+universes.append(Universe(name='mox-4.3% pin cell'))
+universes.append(Universe(name='mox-7% pin cell'))
+universes.append(Universe(name='mox-8.7% pin cell'))
+universes.append(Universe(name='guide tube'))
+universes.append(Universe(name='fission chamber'))
+universes.append(Universe(name='moderator'))
+universes.append(Universe(name='assembly 1'))
+universes.append(Universe(name='assembly 2'))
+universes.append(Universe(name='semi-finely spaced moderator'))
+universes.append(Universe(name='right reflector'))
+universes.append(Universe(name='bottom corner reflector'))
+universes.append(Universe(name='bottom reflector'))
+root = Universe(name='root universe')
+
+universes[0].addCell(cells[0])
+universes[0].addCell(cells[1])
+universes[0].addCell(cells[2])
+universes[0].addCell(cells[3])
+universes[1].addCell(cells[4])
+universes[1].addCell(cells[5])
+universes[1].addCell(cells[6])
+universes[1].addCell(cells[7])
+universes[2].addCell(cells[8])
+universes[2].addCell(cells[9])
+universes[2].addCell(cells[10])
+universes[2].addCell(cells[11])
+universes[3].addCell(cells[12])
+universes[3].addCell(cells[13])
+universes[3].addCell(cells[14])
+universes[3].addCell(cells[15])
+universes[4].addCell(cells[16])
+universes[4].addCell(cells[17])
+universes[4].addCell(cells[18])
+universes[4].addCell(cells[19])
+universes[5].addCell(cells[20])
+universes[5].addCell(cells[21])
+universes[5].addCell(cells[22])
+universes[5].addCell(cells[23])
+universes[6].addCell(cells[24])
+universes[7].addCell(cells[25])
+universes[8].addCell(cells[26])
+universes[9].addCell(cells[27])
+universes[10].addCell(cells[28])
+universes[11].addCell(cells[29])
+universes[12].addCell(cells[30])
+root.addCell(cells[31])
```

Comment:
My opinion would be to add cells as you create universes, rather than this unintelligible and long list of `addCell` calls after the fact.

Even something like
```
universes.append(Universe(name='uo2 pin cell'))
universes[-1].addCell(cells[0])
universes[-1].addCell(cells[1])
```